### PR TITLE
Update d8 to latest

### PR DESCRIPTION
**Pull request by kageiit**

### CODE REVIEW

**Comment by kageiit** [comment]
cc @styurin @asp2insp

File: build.xml [hunk 1]
```diff
@@ -179,6 +179,7 @@
     <include name="jimfs/jimfs-1.1.jar" />
     <include name="jopt-simple/jopt-simple-4.6.jar" />
     <include name="json-simple/json-simple-1.1.1.jar" />
+    <include name="kotlinx-metadata-jvm/kotlinx-metadata-jvm-0.0.3.jar" />
```
Comment (kageiit) [inline build.xml]: This is only required for d8 runtime so I did not add it to idea project files

**Comment by styurin** [comment]
Could you add readme to indicate where that library was downloaded from?

**Comment by kageiit** [comment]
@styurin done

**Pull request closed without merging**